app/include/zmk/event-manager.h

```diff
@@ -78,6 +78,10 @@ struct zmk_event_subscription {
 #define ZMK_EVENT_RELEASE(ev) \
     zmk_event_manager_release((struct zmk_event_header *)ev);
 
+#define ZMK_EVENT_RELEASE_AGAIN(ev) \
+    zmk_event_manager_release_again((struct zmk_event_header *)ev);
+
 int zmk_event_manager_raise(struct zmk_event_header *event);
 int zmk_event_manager_raise_after(struct zmk_event_header *event, const struct zmk_listener *listener);
 int zmk_event_manager_release(struct zmk_event_header *event);
+int zmk_event_manager_release_again(struct zmk_event_header *event);
```

Comment:
I'm not sure I love the name here... Am I understanding the intent here to be to have this event start getting processed again, but starting at the handler that previously swallowed it? Giving it a chance to handle it again, and otherwise, let it flow to the next handler(s)?

If that's the intent, perhaps some thing like "reprocess_from_capture" or "reraise_at_capture". I don't love those names, but I certainly find them to be slightly clearer than "release again" to describe this.

Reply:
I struggled with the name too, and don't like it either. Maybe "zmk_event_manager_raise_before" which is used similar to "zmk_event_manager_raise_after" but raises the event just before the current handler?